tests/integration/cartography/intel/aws/ec2/test_ec2_instances.py

```diff
@@ -90,3 +93,87 @@ def test_ec2_reservations_to_instances(neo4j_session, *args):
         for n in nodes
     }
     assert actual_nodes == expected_nodes
+
+
+def test_ec2_iaminstanceprofiles(neo4j_session, *args):
+    """
+    Ensure that iaminstanceprofiles are connected to their expected iam roles
+    """
+    neo4j_session.run(
+        """
+        MERGE (aws:AWSAccount{id: {aws_account_id}})
+        ON CREATE SET aws.firstseen = timestamp()
+        SET aws.lastupdated = {aws_update_tag}
+        """,
+        aws_account_id="OWNER_ACCOUNT_ID",
```

Comment:
I think this is the source of the bug you mentioned. Change `"OWNER_ACCOUNT_ID"` to `TEST_ACCOUNT_ID`.

Reply:
I need the account id to match the ones in the existing instances from previous tests. I didn't get the memo that `TEST_ACCOUNT_ID` was the same as the ID of AWSAccount. Do I have to l`oad_ec2_instances()` for each individual AWSAccount ID?

Reply:
I bring up that specific variable because it's available to you in the test so you might as well use it: https://github.com/lyft/cartography/blob/342c21ba622872651313989713f7d209006967f0/tests/integration/cartography/intel/aws/ec2/test_ec2_instances.py#L7

I don't think this particular test needs more than one AWS account so we can just use this single ID.

Reply:
Okay, sounds good. I originally did those two strings because they were what was in the existing instances used for the other tests (among the other poorly named things in those test instances)